rs/registry/admin/src/main.rs

```diff
@@ -1215,6 +1218,41 @@ impl ProposalPayload<SubnetRentalRequest> for ProposeToRentSubnetCmd {
         }
     }
 }
+/// Sub-command to submit a subnet rental request proposal.
+#[derive_common_proposal_fields]
+#[derive(Parser, ProposalMetadata)]
+struct ProposeToFulfillSubnetRentalRequestCmd {
+    /// The principal ID of the user whose rental request should be fulfilled
+    #[clap(long)]
+    user: Option<PrincipalId>,
+
+    /// Node IDs that will be members of the subnet
+    #[clap(long, num_args(1..))]
+    node_ids: Option<Vec<PrincipalId>>,
+
+    /// Replica version ID (40 character hexadecimal git commit ID)
+    #[clap(long)]
+    replica_version_id: Option<String>,
```

Comment:
Why are all fields Option? IIUC, this is not done elsewhere.

Reply:
updated.  This is done sometimes, when it's an optional field.  Followed wrong example it seems :)